subprojects/language-scala/src/main/java/org/gradle/language/scala/tasks/BaseScalaCompileOptions.java

```diff
@@ -31,7 +30,6 @@
 /**
  * Options for Scala platform compilation.
  */
-@Incubating
```

Comment:
This type is also used by the "normal" scala plugin and is thus not deprecated. It should not be incubating.